eos/b-decays/b-to-3l-nu.cc

```diff
@@ -0,0 +1,187 @@
+        return prefactor * amp;
+        }
+
+        double decay_width_5diff(const double & q2, const double & k2, const double & y/*cos(theta_gamma)*/, const double & z/*cos(theta_W)*/, const double & phi) const
+        {
+            const complex<double> F_1 = form_factors->F_1(q2, k2);
+            const complex<double> F_2 = form_factors->F_2(q2, k2);
+            const complex<double> F_3 = form_factors->F_3(q2, k2);
+            const complex<double> F_4 = form_factors->F_4(q2, k2);
+
+            return 1.0;//Implement the amplitude like: power_of<2>(g_fermi * abs(model->ckm_ub() ) * abs(cVL) ) * alpha_qed *  (32.0 * M_PI) [...] ;
+        }
+
+        double branching_ratio_2diff(const double & q2, const double & k2) const
+        {
+            return decay_width_2diff(q2, k2) * tau_B / hbar;
+        }
+
+        double branching_ratio_5diff(const double & q2, const double & k2, const double & y/*cos(theta_gamma)*/, const double & z/*cos(theta_W)*/, const double & phi) const
+        {
+            return decay_width_5diff(q2, k2, y, z, phi) * tau_B / hbar;
+        }
+    };
+
+    const std::vector<OptionSpecification>
+    Implementation<BToThreeLeptonsNeutrino>::options
+    {
+        { "lprime", { "e", "mu" }, "mu" },
+        { "l", { "e", "mu" }, "e" },
+    };
+
+    BToThreeLeptonsNeutrino::BToThreeLeptonsNeutrino(const Parameters & parameters, const Options & options) :
+        PrivateImplementationPattern<BToThreeLeptonsNeutrino>(new Implementation<BToThreeLeptonsNeutrino>(parameters, options, *this))
+    {
+    }
+
+    BToThreeLeptonsNeutrino::~BToThreeLeptonsNeutrino()
+    {
+    }
+
+    double
+    BToThreeLeptonsNeutrino::decay_width_2diff(const double & q2, const double & k2) const
```

Comment:
For the externally-visible methods, can we try to follow what is done in other decays? Please check what ``BToPiPiLeptonNeutrino`` does.